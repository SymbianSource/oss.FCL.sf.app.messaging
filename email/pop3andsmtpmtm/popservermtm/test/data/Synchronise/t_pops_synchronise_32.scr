+OK demon POP3 server ready (demon.co.uk)
+OK 
+OK 
+OK 12 18499
+OK 12 messages
1 1519
2 1598
3 1599
4 1559
5 1355
6 1519
7 1519
8 1598
9 1557
10 1558
11 1519
12 1599
.
+OK 12 messages
1 t_pops_synchronise:message026
2 t_pops_synchronise:message027
3 t_pops_synchronise:message028
4 t_pops_synchronise:message029
5 t_pops_synchronise:message030
6 t_pops_synchronise:message031
7 t_pops_synchronise:message032
8 t_pops_synchronise:message033
9 t_pops_synchronise:message034
10 t_pops_synchronise:message035
11 t_pops_synchronise:message036
12 t_pops_synchronise:message037
.
-ERR Unknown Command "CAPA"
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message027; Tue, 14 Aug 2001 16:42:59 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id am2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e5d375c@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:43:07 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHAK9>; Tue, 14 Aug 2001 16:45:58 +0100
Message-ID: <t_pops_synchronise_message027@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 27
Date: Tue, 14 Aug 2001 16:38:08 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Twenty-Seven Twenty-Seven Twenty-Seven Twenty-Seven Twenty-Seven
.
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message026; Tue, 14 Aug 2001 16:10:55 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id af2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e5cedd0@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:42:48 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHAK8>; Tue, 14 Aug 2001 16:45:35 +0100
Message-ID: <t_pops_synchronise_message026@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 26
Date: Tue, 14 Aug 2001 16:37:47 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Twenty-six Twenty-six Twenty-six Twenty-six Twenty-six Twenty-six Twenty-six
.
** Read cancelled
+OK 
