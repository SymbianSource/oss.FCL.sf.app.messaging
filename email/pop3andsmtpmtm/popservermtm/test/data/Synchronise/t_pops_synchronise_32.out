STAT
LIST
UIDL
CAPA
TOP 2 1
TOP 1 1
QUIT
